drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/RULEScenarioValidation.java

```diff
@@ -48,11 +49,12 @@
      * - navigation of bean still valid
      * - field type changed
      * @param simulation
+     * @param settings
      * @param kieContainer
      * @return
      */
     @Override
-    public List<FactMappingValidationError> validate(Simulation simulation, KieContainer kieContainer) {
+    public List<FactMappingValidationError> validate(Simulation simulation, Settings settings, KieContainer kieContainer) {
```

Comment:
I noticed you added settings parameter but do not use it. But if we will set rule group for test scenario background would the rule group validated on this step?  Shouldn't we include the validation of settings here?

Reply:
@danielezonca 
As far as I remember there was not the validation of any "setting: we could add one, but I would do in another ticket.
Beside that, that "Settings" param is there only to fullfill the contract of the declared abstract method